### DISCUSSION TITLE
Usage of about layout template

### DISCUSSION THREAD

**Discussion by moebiusband73** [thuliteio/doks#1123] [General]
Hi,
in the layout about/single page (in core-doks) there is a placeholder [image]. I am not sure how to populate it?
It is not inside a Go template statement, so I am unsure how to set an image path there.
Thanks for your help!
Jan

**Comment by d3b0o** [comment]
I have the same problem, did yo solve it?

**Comment by moebiusband73** [comment]
Hi,
I think I overwrote the about layout. So I put this code into `./layouts/about/single.html `:
```
{{ define "main" }}
<div class="row row-about justify-content-center align-items-center">
  <div class="col-md-16 col-lg-7">
    <article>
      <h1>{{ .Title }}</h1>
      {{ .Content }}
    </article>
  </div>
  <div class="col-md-16 col-lg-7 text-center">
    {{ partial "utilities/image-fluid" "images/cc-arch.png" }}
  </div>
</div>
{{ end }}
```

You need to to specify there how you want this part to look like. The content part is defined in` ./data/about.yaml` .
Hope this helps. I think it would be a good idea for doks to just be a fancy documentation theme for hugo and not to try  to hide the Hugo madness :-). I guess to try is a lost battle.
Best, Jan
